### ISSUE TITLE
Can we get back in sync? Update JUCE, get it to compile on Mac, and additional changes needed

### ISSUE THREAD

**Issue by elanhickler** [RobinSchmidt/RS-MET#274]
1. Update JUCE
2. Get Mac compile working
3. unprotect `virtual double openModalNumberEntryField(double numberToShowInitially);`
4. unprotect `virtual void showAutomationSetup();`
5. unprotect `virtual void showModulationSetup();`

6. make Async Updater safer and more correctly implemented

![image](https://user-images.githubusercontent.com/9116575/56095994-3f80f780-5e97-11e9-99a6-41fb854a7c83.png)

![image](https://user-images.githubusercontent.com/9116575/56096000-47409c00-5e97-11e9-9ce4-ed468ea5b456.png)

![image](https://user-images.githubusercontent.com/9116575/56096005-59223f00-5e97-11e9-81dd-8cc552c4df96.png)

**Comment by RobinSchmidt** [comment]
ok - but if you don't mind, i'd like to prepare the next invoice before. what's wrong with my repaintOnMessageThread? i mean, of course, aside from being admittedly somewhat quirky and unelegant. but why would it be considered unsafe? i think, it may be a leaner implementation compared to adding yet another baseclass (which could perhaps be saved for other stuff)

**Comment by elanhickler** [comment]
This was suggested by Jim to solve some crashes I am having with my plugins. I think it helped but ultimately still getting crashes. I've been waiting for us to get back in sync so you could try to fix these crashes. It's a problem of the UI updating when some pointers become null.

**Comment by elanhickler** [comment]
why no **progresssss**

**Comment by RobinSchmidt** [comment]
yes - i guess, i was just waiting from some notice like ..."sure, drop me the invoice" or something. but i also lost some steam, got out of step. i think, my productivity is sometimes a bit like a square-wave

**Comment by elanhickler** [comment]
increase your duty cycle frequency, jeezmus!

**Comment by gregjazz** [comment]
Sure, drop me the invoice! :)

**Comment by elanhickler** [comment]
stop creating new code and upgrade juuuuuuuuceeeeeeeeeeeeeeee plzzz. This weekend I want to work on Soundemote plugins, can't do it until you upgrade.

**Comment by RobinSchmidt** [comment]
ok - i can take a break from sync now. i have it now in a state, where i can pick it up later without too much interruption loss. ...i have now a unit test in place that exactly exposes where my current code goes wrong - thinking about the solution can come later. i've send you an email with my updated sheet of work times (and also another one for the sine-modeling - but that is for greg anyway)

**Comment by RobinSchmidt** [comment]
ok - i'm in the process of updating (it's on a new branch - but when done, i'll merge to master)

**Comment by elanhickler** [comment]
![image](https://user-images.githubusercontent.com/9116575/57170920-f7cdfb80-6dc4-11e9-8811-aa8ae23c14e0.png)
👍

**Comment by elanhickler** [comment]
I'm on your branch, FlowerChildFilter builds after commenting out a series of errors I don't yet know how to solve on my code base. And unprotecting a few of your class members by changing your library code on my end

**Comment by RobinSchmidt** [comment]
>  sometimes the modules will not work/register in the .jucer project unless you fully remove the module then re-add.

i had the same problem yesterday. i thought (after a lot of banging my head against the wall), this was due to some (strange) change in the projucer, to not allow custom juce modules residing in the same folder as the normal juce modules anymore:

https://forum.juce.com/t/projucer-5-4-1-doesnt-allow-user-modules-to-be-inside-juce-modules-folder/30363

however, just moving them into another folder and trying to change the paths - it still didn't work. only after a lot more headbanging, i removed and re-added the modules - and then it magically did work - without rhyme or reason. these are the kinds of things that are *really* *REALLY* frustrating. something doesn't work and you have to make all sorts of guesses how it possibly could be fixed

**Comment by RobinSchmidt** [comment]
the branch is now merged into the master branch. by the way. as predicted, building vst2 plugins doesn't work out the box anymore - you need to somewhere have the vst2 sdk. i added a BuildNotes.txt file to the ToolChain project to explain, what has to be done to build it for vst2. a big disimprovement ;-(  now, the main project of the codebase will not build anymore without jumping through hoops like obtaining the vst2 sdk somewhere and putting it into the right place or installing the vst3 sdk and reconfiguring the jucer file ...at least, the standalone can still be built out of the box - but ToolChain standalone is not really of much value aside from testing. but i guess, roli is not to blame for this mess - it's steinberg's bitchiness with regard to redistribution of the vst sdk

**Comment by RobinSchmidt** [comment]
oh - i didn't see the String::empty warnings. oh - i see - probably they occur only at a higher warning level than i use (i normally use medium for debug builds and high only for release builds which i rarely do)

**Comment by elanhickler** [comment]
please fix your warnings, 

![image](https://user-images.githubusercontent.com/9116575/57182182-3f01ce00-6e51-11e9-814e-f601e14b58f8.png)

If you want me to do it, I can. Do you want {} or Class()?

```
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_elan_juce_helpers.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\widgets\jura_RWidget.h	84	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\FlowerChildFilter.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\widgets\jura_RWidget.h	84	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_elan_juce_helpers.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\audio\jura_AudioPlugin.h	26	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_elan_juce_helpers.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\audio\jura_AudioPlugin.h	110	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\FlowerChildFilter.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\audio\jura_AudioPlugin.h	26	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\FlowerChildFilter.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\audio\jura_AudioPlugin.h	110	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_elan_juce_helpers.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\widgets\jura_RTextField.h	254	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\FlowerChildFilter.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\widgets\jura_RTextField.h	254	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\FlowerChildFilter.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\widgets\jura_RTextField.h	18	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\widgets\jura_RWidget.h	84	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\audio\jura_AudioPlugin.h	26	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\audio\jura_AudioPlugin.h	110	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\tools\jura_StringTools.cpp	207	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\tools\jura_StringTools.cpp	255	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\tools\jura_StringTools.cpp	410	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\tools\jura_XmlTools.cpp	27	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\control\jura_Parameter.cpp	330	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\control\jura_Parameter.cpp	334	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\control\jura_Parameter.cpp	343	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\control\jura_Parameter.cpp	351	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\control\jura_Parameter.cpp	359	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\control\jura_Parameter.cpp	366	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\control\jura_MetaParameter.cpp	345	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\control\jura_MetaParameter.cpp	466	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\control\jura_ModulatableParameter.cpp	650	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\control\jura_StateManager.cpp	23	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\control\jura_StateManager.cpp	141	
Warning	C4996	'juce::File::nonexistent': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\files\jura_FileManager.cpp	44	
Warning	C4996	'juce::File::nonexistent': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\files\jura_FileManager.cpp	49	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\files\jura_FileManager.cpp	84	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\files\jura_FileManager.cpp	240	
Warning	C4996	'juce::File::nonexistent': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\files\jura_FileManager.cpp	285	
Warning	C4996	'juce::File::nonexistent': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\files\jura_FileManager.cpp	337	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\files\jura_FileManager.cpp	371	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\files\jura_StateFileManager.cpp	71	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\files\jura_ImageFileManager.cpp	8	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\files\jura_ImageFileManager.cpp	9	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\graphics\jura_ColorMap.cpp	73	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\graphics\jura_ColorMap.cpp	74	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\graphics\jura_ColourScheme.cpp	20	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\graphics\jura_GraphicsTools.cpp	1135	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\misc\jura_DescribedComponent.cpp	26	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\misc\jura_DescribedComponent.cpp	53	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\widgets\jura_RTextField.cpp	308	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\widgets\jura_RTextEditor.cpp	95	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\widgets\jura_RTextEditor.cpp	1363	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\widgets\jura_RButton.cpp	12	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\widgets\jura_RTreeView.h	23	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\widgets\jura_RPopUpMenu.cpp	33	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\widgets\jura_RPopUpMenu.cpp	125	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\widgets\jura_RComboBox.cpp	163	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\widgets\jura_RTextField.h	254	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\widgets\jura_RComboBox.h	44	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\widgets\widget_sets\jura_StateLoadSaveWidgetSet.cpp	11	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\widgets\widget_sets\jura_FileSelectionBox.cpp	14	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\widgets\jura_RWidget.h	84	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\plots\jura_FunctionPlot.cpp	28	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\plots\jura_FunctionPlot.cpp	31	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\plots\jura_SpectrumPlot.cpp	15	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\plots\jura_SpectrumPlot.cpp	19	
Warning	C4996	'juce::File::nonexistent': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\files\jura_AudioFileInfo.h	24	
Warning	C4996	'juce::File::nonexistent': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\audio\jura_AudioFileBuffer.cpp	9	
Warning	C4996	'juce::File::nonexistent': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\audio\jura_AudioFileBuffer.cpp	131	
Warning	C4996	'juce::File::nonexistent': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\audio\jura_AudioFileBufferUser.cpp	93	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\audio\jura_AudioFileBufferUser.cpp	102	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\audio\jura_AudioFileBufferUser.cpp	111	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\panels\jura_CoordinateSystem.cpp	8	
Warning	C4996	'juce::File::nonexistent': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\panels\jura_CoordinateSystem.cpp	850	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\panels\jura_CoordinateSystem.cpp	915	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\audio\jura_AudioPlugin.h	26	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\audio\jura_AudioPlugin.h	110	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\panels\jura_CurveFamilyPlot.cpp	342	
Warning	C4996	'juce::File::nonexistent': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\panels\jura_InteractiveCoordinateSystem.cpp	249	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\dialogs\jura_ColourSchemeSetupDialog.cpp	185	
Warning	C4996	'juce::File::nonexistent': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\dialogs\jura_ImageSavingDialog.cpp	99	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\dialogs\jura_ImageSavingDialog.cpp	138	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\dialogs\jura_ImageSavingDialog.cpp	236	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\audio\jura_AudioModule.cpp	403	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\widgets\jura_RTextField.h	18	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\audio\jura_AudioModuleFactory.cpp	11	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\widgets\jura_RTextField.h	254	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\widgets\jura_RTreeView.h	23	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\basics\jura_RoutingMatrix.cpp	92	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\basics\jura_RoutingMatrix.cpp	97	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\analyzers\jura_MidiMonitor.cpp	340	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\analyzers\jura_MidiMonitor.cpp	344	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\widgets\jura_RComboBox.h	44	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\analyzers\jura_MultiAnalyzer.cpp	226	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\analyzers\jura_MultiAnalyzer.cpp	417	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\analyzers\jura_MultiAnalyzer.cpp	418	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\modulators\jura_BreakpointModulatorAudioModule.cpp	81	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\modulators\jura_BreakpointModulatorAudioModule.cpp	176	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\modulators\jura_BreakpointModulatorAudioModule.cpp	193	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\modulators\jura_BreakpointModulatorAudioModule.cpp	213	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\modulators\jura_ModulatorCurveEditor.cpp	25	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\modulators\jura_ModulatorCurveEditor.cpp	29	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\widgets\jura_RTextField.h	18	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\modulators\jura_BreakpointModulatorEditor.cpp	319	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\generators\jura_OscillatorStereo.cpp	69	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\generators\jura_SamplePlayer.cpp	204	
Warning	C4996	'juce::File::nonexistent': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\audio\jura_AudioFileBuffer.h	34	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\generators\jura_SamplePlayer.cpp	411	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\generators\jura_SamplePlayer.cpp	412	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\generators\jura_SamplePlayer.cpp	1170	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\widgets\jura_RTextEditor.h	63	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	171	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	178	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	185	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	192	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	199	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	206	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	213	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	220	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	227	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	234	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	241	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	248	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	255	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	262	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	269	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	276	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	283	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	290	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	297	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	304	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	311	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	318	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	325	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	332	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	339	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	346	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	353	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	360	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	367	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	374	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	381	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	388	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	395	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	402	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	409	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	416	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	423	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	454	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	464	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	474	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	484	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	494	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	504	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	514	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	524	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	534	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	544	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	554	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	564	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	574	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	584	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	594	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	604	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	614	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	624	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	634	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	644	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	654	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	664	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	674	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	684	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	694	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	704	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	714	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	724	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	734	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	744	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	754	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	764	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	774	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	784	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	794	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	804	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\effects\jura_Quadrifex.cpp	814	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\instruments\jura_Straightliner.cpp	66	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\instruments\jura_Straightliner.cpp	70	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\instruments\jura_Straightliner.cpp	74	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\instruments\jura_Straightliner.cpp	78	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\instruments\jura_Straightliner.cpp	80	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\instruments\jura_SimpleSampler.cpp	65	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\instruments\jura_KeyShot.cpp	51	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\instruments\jura_Quadrigen.cpp	82	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\instruments\jura_Quadrigen.cpp	105	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\instruments\jura_Liberty.cpp	145	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\instruments\jura_Liberty.cpp	204	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\instruments\jura_Liberty.cpp	206	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\instruments\jura_Liberty.cpp	213	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\widgets\jura_RTextField.h	123	
Warning	C4244	'argument': conversion from 'double' to 'int', possible loss of data (compiling source file ..\..\JuceLibraryCode\include_rosic.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\rosic\unfinished\rosic_ModalSynth.cpp	346	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_jura_processors.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_processors\misc\jura_DspWorkbench.cpp	258	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_se_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\widgets\jura_RWidget.h	84	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_se_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\audio\jura_AudioPlugin.h	26	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_se_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\audio\jura_AudioPlugin.h	110	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\Source\FlowerChildFilterCore.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\widgets\jura_RWidget.h	84	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_se_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\widgets\jura_RTextField.h	254	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\Source\FlowerChildFilterCore.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\audio\jura_AudioPlugin.h	26	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\Source\FlowerChildFilterCore.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\audio\jura_AudioPlugin.h	110	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\Source\FlowerChildFilterEditor.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\widgets\jura_RWidget.h	84	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\Source\FlowerChildFilterCore.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\widgets\jura_RTextField.h	254	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\Source\FlowerChildFilterCore.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\widgets\jura_RTextField.h	18	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\Source\FlowerChildFilterEditor.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\audio\jura_AudioPlugin.h	26	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\Source\FlowerChildFilterEditor.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\audio\jura_AudioPlugin.h	110	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_se_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\widgets\jura_RTextField.h	18	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\Source\FlowerChildFilterEditor.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\widgets\jura_RTextField.h	254	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\Source\FlowerChildFilterEditor.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\widgets\jura_RTextField.h	18	
Warning	C4996	'juce::File::nonexistent': was declared deprecated (compiling source file ..\..\JuceLibraryCode\include_se_framework.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\Soundemote\Shared\se_framework\ElanSynthLib\PluginComponents\BasicEditor.cpp	315	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\Source\FlowerChildFilterModule.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\widgets\jura_RWidget.h	84	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\Source\FlowerChildFilterModule.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\audio\jura_AudioPlugin.h	26	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\Source\FlowerChildFilterModule.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\audio\jura_AudioPlugin.h	110	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\Source\FlowerChildFilterModule.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\widgets\jura_RTextField.h	254	
Warning	C4996	'juce::String::empty': was declared deprecated (compiling source file ..\..\Source\FlowerChildFilterModule.cpp)	Flower Child Filter_SharedCode	D:\_PROGRAMMING\RS-MET\Libraries\RobsJuceModules\jura_framework\gui\widgets\jura_RTextField.h	18	
```

**Comment by RobinSchmidt** [comment]
done

**Comment by elanhickler** [comment]
`  if( fileToLoadFrom != File() )`

robin, yyyy? Wouldn't you use: `fileToLoadFrom.existsAsFile()` 

Also, `if( fileToLoadFrom != File() )` would potentially return true if it exists as a folder. So using that boolean is safer.

**Comment by elanhickler** [comment]
solved

**Comment by RobinSchmidt** [comment]
hmm...dunno. juce itself sometimes does it like that:
```cpp
URL::URL (File localFile)
{
    if (localFile == File())
        return;
```
i guess, i just copy-and-pasted or something. i think, a directory is also not equivalent to a nonexistent file, so if it's a dir, it will also not be equal to File(). probably doesn't make a difference. however, i agree that your version looks nicer, so i changed it

**Comment by RobinSchmidt** [comment]
oh - what about your puul request? is that obsolete now? i think, you requested a few more changes, but it's probably more convenient, if i just apply them manually?

**Comment by elanhickler** [comment]
cancel that one.
